results/pkg/api/server/test/db.go

```diff
@@ -0,0 +1,56 @@
+package test
```

Comment:
In other tekton projects, we've been following the pattern of putting all test helpers/e2e tests in a separate test/ folder that is in the same level as pkg/

Reply:
I'm not sure we want to do that here - I suspect that we may want to split the API Server / watcher into separate modules in the future, so keeping this closer to the rest of the API Server code will make that transition easier if/when that happens.